extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -106,10 +106,7 @@ export class AmpSelector extends AMP.BaseElement {
     this.init_();
     if (!this.isDisabled_) {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
-      if (this.kbSelectMode_ != KEYBOARD_SELECT_MODES.NONE) {
```

Comment:
`touch` does not mean `!keyboard`. Folks with dexterity use physical keyboard on their smartphones. We should always assume a physical keyboard exist. Adding a `listener` that may not be called is not something to optimize for anyway. It should have ~0 perf impact.